setup-prometheus.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd kube-prometheus
+./monitoring-deploy.sh
+cd ..
```

Comment:
We should be able to include this little script directly in the other setup script, shouldn't we?

Reply:
Done, I was not aware how to deal with the relative file paths within the invoked script, I have fixed it now.